test/Tool/saturating1.opt

```diff
@@ -0,0 +1,10 @@
+; REQUIRES: solver
+
+; RUN: %souper-check %solver -infer-rhs -print-replacement %s > %t1
+; RUN: %FileCheck %s -check-prefix=SUCCESS < %t1
+
+; SUCCESS: RHS inferred successfully
```

Comment:
do you want to look for a particular RHS? or are you just testing that souper thinks it has succeeded here?

Reply:
yeah, this test is silly. I will remove it. I think we have enough tests for this one now.

Reply:
fixed